lib/osv/impact.py

```diff
@@ -163,7 +167,8 @@ def get_affected_range(repo, regress_commits, fix_commits):
       if equivalent_regress_commit:
         break
 
-    if not equivalent_regress_commit:
+    if not equivalent_regress_commit and regress_commits:
```

Comment:
This condition is unclear, why add regress_commits ? maybe add more in comment.

Reply:
I did add comment here but I'll clarify it a bit more.